docs/theme.config.tsx

```diff
@@ -80,19 +89,22 @@ const config: DocsThemeConfig = {
   head: function Head() {
     const router = useRouter();
     const { systemTheme = "dark" } = useTheme();
-    const { frontMatter } = useConfig();
+    const nextraConfig = useConfig();
+
+    const frontMatter = nextraConfig.frontMatter as Frontmatter;
     const fullUrl =
       router.asPath === "/" ? SITE_ROOT : `${SITE_ROOT}${router.asPath}`;
 
     const asPath = router.asPath;
 
-    let ogUrl;
+    let ogUrl: string;
 
     if (asPath === "/") {
       ogUrl = `${SITE_ROOT}/api/og`;
-    } else if (frontMatter?.ogImage) {
+    } else if (frontMatter.ogImage) {
       ogUrl = `${SITE_ROOT}${frontMatter.ogImage}`;
     } else {
+      // eslint-disable-next-line no-nested-ternary -- This is fine.
```

Comment:
👎🏾

Reply:
Good catch, forgot I left this in. It was this connotation:

![image](https://github.com/vercel/turbo/assets/35677084/4ac5096b-b4ac-4087-8305-73cf0243d642)

Reply:
haha i didn't catch that. i am very bad at reading ternaries